dox/books/reference_manual/DoxyRefman

```diff
@@ -753,7 +753,9 @@ WARN_LOGFILE           =
 # spaces.
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = dox/reference_manual \
+INPUT                  = dox/books/reference_manual \
+                         src/compiler \
```

Comment:
Why are you including the src folders here?

Reply:
Reference manual should be based on the source code.